src/flownet/ert/_create_ert_setup.py

```diff
@@ -230,17 +296,25 @@ def create_ert_setup(  # pylint: disable=too-many-arguments
             # Otherwise create an empty one.
             (output_folder / f"{section}.inc").touch()
 
+    ert_output_file_names = os.path.join(output_folder, "observations.ertobs")
+
+    yaml_output_file_names = os.path.join(output_folder, "observations.yamlobs")
+
     if not prediction_setup:
         if parameters is not None:
             _create_observation_file(
                 schedule,
-                output_folder / "observations.ertobs",
+                ert_output_file_names,
```

Comment:
If you revert back to `pathlib` you can do this type of operation again, which the standard/preferred way of doing operations with paths used in FlowNet.